python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/dagster_sling_translator.py

```diff
@@ -0,0 +1,205 @@
+import re
+from typing import Any, Iterable, Mapping, Optional
+
+from dagster import (
+    AssetKey,
+    AutoMaterializePolicy,
+    FreshnessPolicy,
+    MetadataValue,
+)
+from dagster._annotations import public
+
+
+class DagsterSlingTranslator:
```

Comment:
Would it make sense to make this a dataclass so we don't need to write the constructor?